scripts/dockerlib.sh

```diff
@@ -44,6 +45,9 @@ LOG_FILE=$LOG_DIR/docker-handler.log
 CONFIG_FILE=$(ls $CONFIG_DIR | grep -E ^[0-9]+.settings$ | sort -n | tail -n 1)
 STATUS_FILE=$(echo $CONFIG_FILE | sed s/settings/status/)
 
+CONFIG=$CONFIG_DIR/$CONFIG_FILE
```

Comment:
According to this (http://stackoverflow.com/questions/673055/correct-bash-and-shell-script-variable-capitalization) all the internal variables we use should be lowercase.  We should only make those variables we export to the environment all uppercase.  This is to avoid accidental blocking of environment variable values and seems a decent convention.  I'll change all these variables to lower case.